components/collective-navbar/menu.js

```diff
@@ -70,7 +71,7 @@ const addSectionLink = (intl, links, collective, sections, section) => {
  */
 const getCategoryMenuLinks = (intl, collective, sections, category) => {
   const links = [];
-  const collectiveSlug = collective.slug;
+  const collectiveSlug = getCollectivePageRoute(collective);
```

Comment:
Variable name seems outdated since this does not contain a slug anymore

Reply:
Renamed to `collectivePageRoute`.